src/monero_types.h

```diff
@@ -21,18 +21,11 @@
 
 #include "os_io_seproxyhal.h"
 
-#if CX_APILEVEL == 8
-#define PIN_VERIFIED (!0)
-#elif CX_APILEVEL >= 9
-#define PIN_VERIFIED BOLOS_UX_OK
-#else
-#error CX_APILEVEL not  supported
-#endif
-
-/* cannot send more that F0 bytes in CCID, why? do not know for now
- *  So set up length to F0 minus 2 bytes for SW
+/* The maximal APDU length is 0xF0 bytes, including the 2 bytes needed for the SW response,
```

Comment:
```suggestion
/* The maximal APDU response length is 0xF0 bytes, including the 2 bytes needed for the SW response,
```
and do you now why? Or we are not talking about the response? all this is confusing.